releases/v9.0.0.rst

```diff
@@ -0,0 +1,122 @@
+
+PostgREST version 9.0
+=====================
+
+This mayor version is released following the PostgreSQL 14 launch and is accompanied with new features, improvements to functionality and bug fixes. Download the new version in the `PostgREST v9.0.0 release page <https://github.com/PostgREST/postgrest/releases>`_.
+
+PostgreSQL 14 Compatibility
+---------------------------
+
+The latest PostgreSQL release tightened the GUC naming scheme making it impossible to use dots (``.``) and dashes (``-``). Thankfully, `@robertsosinski <https://github.com/robertsosinski>`_ got the PostgreSQL team to reconsider allowing dots in the GUC name, making it possible to avoid a breaking change. You can see the full discussion `here <https://www.postgresql.org/message-id/17045-6a4a9f0d1513f72b%40postgresql.org>`_.
+
+Since dashes cannot be used on PostgreSQL 14, the GUC names are changed to the ones :ref:`mentioned in this section <guc_req_headers_cookies_claims>`. On older PostgreSQL versions we still use the :ref:`guc_legacy_names`. If you wish to use the new JSON GUCs on these versions, set the :ref:`db-use-legacy-gucs` config option to false.
+
+Embedding on Partitioned Tables (and other features)
+----------------------------------------------------
+
+You can now confidently :ref:`embed partitioned tables <embedding_partitioned_tables>` with other tables; meanwhile, their partitions will stay hidden and won't interfere in the process. Likewise, other supported features for partitioned tables are: UPSERT, INSERT with Location response, OPTIONS request and OpenAPI support.
+
+Dropping support for PostgreSQL 9.5
+-----------------------------------
+
+Due to the recent `end of life of PostgreSQL 9.6 <https://www.postgresql.org/support/versioning/>`_, we are dropping support for the next older version. Take into consideration that we are also planning on dropping the support for 9.6 in the next minor versions.
```

Comment:
```suggestion
Due to the recent `end of life of PostgreSQL 9.6 <https://www.postgresql.org/support/versioning/>`_, we are dropping support for the next older version. Take into consideration that we are also planning on dropping the support for 9.6 in the next major version.
```

I don't think we can drop support in a minor version. That would be a breaking change.